include/TPP/Dialect/Tpp/TppOps.td

```diff
@@ -35,6 +35,7 @@ def TppVNNIOperand : AnyTypeOf<[TppVNNIMemrefInput, AnyFloat]>;
 
 // Tpp operation trait - make sure the operand are broadcastable
 // following Numpy-style broadcast.
+//  https://numpy.org/doc/stable/user/basics.broadcasting.html#general-broadcasting-rules
```

Comment:
Stop adding these URLs, we do not follow them completely, and we'll remove all of those rules soon.

Reply:
I was just moving it. But I can delete it.

Reply:
Better to remove any comment to the numpy broadcast, or it will pass the idea that we support it fully.